docs/configuration/advanced.md

```diff
@@ -25,6 +27,13 @@ uda.taskwarrior-tui.style.calendar.title=black on rgb444
 uda.taskwarrior-tui.task-report.next.filter=$(task show report.next.filter)
 ```
 
+The `uda.taskwarrior-tui.task-report.next.filter` variable defines the default view at program startup. Set this to any preconfigured report (`task reports`), or create your own report in taskwarrior and specify its name here.
+
+## commandline options:
+
+`-c`: specify a config file other than standard `.taskrc`  
```

Comment:
Currently this doesn’t do anything, so we should remove it from the docs.

Reply:
I have updated the PR accordingly :)